rke/k8s_rke_system_images.go

```diff
@@ -1357,10 +1357,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		"v1.14.9-rancher1-1": {
```

Comment:
Is this incorrectly marked for v2.3.3?

Reply:
@superseb is correct that we need to introduce `v1.14.9-rancher1-2` for 2.3.4